Halachic Times/app/src/main/res/values-lt/strings.xml

```diff
@@ -211,5 +211,6 @@
     <string name="reminder_tuesday">Priminti antradien</string>
     <string name="reminder_wednesday">Priminti trečiadienį</string>
     <string name="notification_upcoming_title">Artėjantis laikas kaip pranešimo apie</string>
+    <string name="opinion_gra">Pasak GRA</string>
```

Comment:
why did this move down here from line 24?

Reply:
andriod studio translation editor adds to the end, i didnt catch this one